firebase-firestore/src/main/java/com/google/firebase/firestore/FirebaseFirestore.java

```diff
@@ -816,4 +823,9 @@ void validateReference(DocumentReference docRef) {
   static void setClientLanguage(@NonNull String languageToken) {
     FirestoreChannel.setClientLanguage(languageToken);
   }
+
+  /** @return A set of concrete implementations of {@code MapEncoder} */
```

Comment:
Although not required, mark this as `@NonNull`. 

The doc can be `@return The set of registered {@code MapEncoder}s`

Reply:
I have updated this Java doc. Thank you.